### PR TITLE
Tweak repo-digest:

### PR DESCRIPTION
**Pull request by mberhault**
- run generated html through premailer to inline css
- allows -o for output dir
- output nextsince and digest file to stdout (for easier parsing)

### CODE REVIEW

File: digest.go [hunk 1]
```diff
@@ -241,19 +236,38 @@ func Digest(c *Context, open, closed []*PullRequest) error {
...
 		Closed: sortedClosed,
 	}
+
 	tmpl := template.Must(template.New("digest").Funcs(template.FuncMap{"markDown": markDowner}).Parse(htmlTemplate))
+	buf := new(bytes.Buffer)
+	if err := tmpl.Execute(buf, content); err != nil {
+		return err
+	}
+
+	options := premailer.NewOptions()
```
Comment (spencerkimball) [inline digest.go]: Let's control the premailer through a command line flag. Put add an `--inline-styles` flag which defaults to true and `Context.InlineStyles bool`.

**Comment by spencerkimball** [comment]
LGTM. You'll see I made some changes which slightly conflict with this (you're going to need to merge).

File: digest.go (hunk 1 above)
Comment (mberhault) [inline digest.go]: Done.

**Comment by mberhault** [comment]
rebased and addressed comments. README is up-to-date as well.

**Pull request merged**